sympl/_components/netcdf.py

```diff
@@ -50,11 +51,25 @@ def __init__(
                 This can result in many file open/close operations.
                 Default is to write only when the write() method is
                 called directly.
+            aliases : dict
+                A dictionary used to rename variables to their aliases.
```

Comment:
Replace this docstring with `A dictionary of string replacements to apply to state variable names before saving them in netCDF files.`